### PR TITLE
New Nano editor Cheat Sheet

### PR DESCRIPTION
**Pull request by N0vaPixel**
##### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Nano editor Cheat Sheet`**

###### Description of changes

Introduce a new cheat sheet of the nano editor.

---

Instant Answer Page: https://duck.co/ia/view/nano_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @N0vaPixel

### CODE REVIEW

**Comment by daxtheduck** [comment]
@N0vaPixel Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by tagawa** [comment]
Hi @N0vaPixel. Welcome and thanks for contributing this cheat sheet! I'm a fan of nano so it's great to see this.

The file looks good but for some reason it's not deploying automatically on our beta server for testing. I'll try to see what's happening.

File: share/goodie/cheat_sheets/json/nano.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
+{
+    "id": "nano_cheat_sheet",
+    "name": "Nano editor",
+    "description": "Nano is a text editor for Unix computing systems.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/nano.json]: lets remove the period at the end of description

File: share/goodie/cheat_sheets/json/nano.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    "metadata": {
+        "sourceName": "nano-editor.org",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/nano.json]: " GNU nano " or "Nano Editor" would be enough here

File: share/goodie/cheat_sheets/json/nano.json [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+            {
+                "key": "[Ctrl] [K]",
+                "val": "Cut a line into the clipboard. You can press this repeatedly to cut multiple lines."
+            },
+            {
+                "key": "[Ctrl] [U]",
+                "val": "Uncut text. You can press this repeatedly to past multiple to past multiple time."
+            },
+            {
+                "key": "[BackSpace]",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/nano.json]: remove the brackets since there is only one key

File: share/goodie/cheat_sheets/json/nano.json [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+            },
+            {
+                "key": "[Ctrl] [W]",
+                "val": "Find a word or phrase. Hit Ctrl+R to move into replace mode. Alternatively you can hit Ctrl+T to go to a specific line."
+            }
+        ],
+        "Editing": [
+            {
+                "key": "[Ctrl] [K]",
+                "val": "Cut a line into the clipboard. You can press this repeatedly to cut multiple lines."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/nano.json]: remove the periods at end of val fields

**Comment by N0vaPixel** [comment]
Hi @tagawa . It's a pleasure to enhance duckduckgo instant answer. Here is my first cheat sheet.
So thanks to @jophab, i added the changes.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=nano cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/nano_cheat_sheet_mobile.png?nQae0qMAgu)

**Comment by tagawa** [comment]
Thanks for the quick feedback @jophab.

@N0vaPixel I manually deployed it to the beta server - not sure why it didn't work automatically but it's OK now: https://beta.duckduckgo.com/?q=nano+cheat+sheet&ia=cheatsheet&iax=1
We can use this for testing and if everything's OK, then merge the PR.

**Comment by tagawa** [comment]
It says the beta deployment was requested but it didn't paste the link automatically, so here it is: https://beta.duckduckgo.com/?q=nano%20cheat%20sheet
Testing comments welcome!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=nano cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/nano_cheat_sheet_mobile.png?iqa252ACUU)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=nano cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/nano_cheat_sheet_mobile.png?P7jr3ZXQKh)

**Comment by tagawa** [comment]
This seems to be OK on the beta server and it looks good to me, so merging. Thank you for the contribution @N0vaPixel! 🍰 

If you're looking for more ideas for contributing, we have a table of programming Instant Answers we're trying to create: https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage

**Pull request merged**